protocol/message_persistence.go

```diff
@@ -951,13 +951,14 @@ func (db sqlitePersistence) LatestPendingContactRequestIDForContact(contactID st
 	return id, nil
 }
 
-func (db sqlitePersistence) LatestContactRequestIDs() (map[string]common.ContactRequestState, error) {
-	res := map[string]common.ContactRequestState{}
+func (db sqlitePersistence) LatestContactRequestIDs() ([]LatestContactRequest, error) {
+	res := make([]LatestContactRequest, 0)
+	// we limit to 20 is enough?
```

Comment:
I'm not usually a fan of such comments in the main branch (maybe you would delete it before merge?) because there's no clear action to be taken. Should the next developer one month from now answer it? Should we investigate in a separate issue?

But it's a good question of course. Why not more than 20, say 200? I can imagine users with a larger number of accumulated contact requests to sync would be surprised with the artificial limit. Seems like syncing could handle more than 20 just fine, but I might be missing a bigger picture.